dynamic-realtime/pranjaljain0.md

```diff
@@ -67,5 +67,3 @@
 </a>
 
 ![visitors](https://visitor-badge.glitch.me/badge?page_id=pranjaljain0.pranjaljain0)
-
-⭐️ From [pranjaljain0](https://github.com/pranjaljain0)
```

Comment:
Well, you need this one
Add it back again.

Reply:
isn't it already added?
damn, I must have confused it,
I'll add again